soh/soh/Enhancements/mods.cpp

```diff
@@ -6,14 +6,29 @@
 extern "C" {
 #include <z64.h>
 #include "macros.h"
+#include "functions.h"
 #include "variables.h"
 extern SaveContext gSaveContext;
 extern PlayState* gPlayState;
 extern void Play_PerformSave(PlayState* play);
 extern s32 Health_ChangeBy(PlayState* play, s16 healthChange);
 extern void Rupees_ChangeBy(s16 rupeeChange);
+extern void Inventory_ChangeEquipment(s16 equipment, u16 value);
```

Comment:
If you're already including the functions.h header here, you shouldn't need the seperate extern functions anymore.